MediaBrowser.Model/Entities/PersonType.cs

```diff
@@ -1,48 +1,68 @@
 namespace MediaBrowser.Model.Entities
 {
     /// <summary>
-    /// Struct PersonType.
+    /// Types of persons.
     /// </summary>
-    public class PersonType
+    public static class PersonType
     {
         /// <summary>
-        /// The actor.
+        /// A person whose profession is acting on the stage, in films, or on television.
         /// </summary>
         public const string Actor = "Actor";
 
         /// <summary>
-        /// The director.
+        /// A person who supervises the actors and other staff in a film, play, or similar production.
         /// </summary>
         public const string Director = "Director";
 
         /// <summary>
-        /// The composer.
+        /// A person who writes music, especially as a professional occupation.
         /// </summary>
         public const string Composer = "Composer";
 
         /// <summary>
-        /// The writer.
+        /// A writer of a book, article, or document. Can also be used as a generic term for music writer if there is a lack of specificity.
         /// </summary>
         public const string Writer = "Writer";
 
         /// <summary>
-        /// The guest star.
+        /// A well-known actor or other performer who appears in a work in which they do not have a regular role.
         /// </summary>
         public const string GuestStar = "GuestStar";
 
         /// <summary>
-        /// The producer.
+        /// A person responsible for the financial and managerial aspects of the making of a film or broadcast or for staging a play, opera, etc.
         /// </summary>
         public const string Producer = "Producer";
 
         /// <summary>
-        /// The conductor.
+        /// A person who directs the performance of an orchestra or choir.
         /// </summary>
         public const string Conductor = "Conductor";
 
         /// <summary>
         /// The lyricist.
         /// </summary>
         public const string Lyricist = "Lyricist";
+
+        /// <summary>
+        /// A person who writes the words to a song or musical.
```

Comment:
"...Reimagines an existing composition, adapting the instruments, voices, rhythms, and tempo to create a new sound for a piece of music."

Reply:
Ah, seems I mis-pasted and pasted the Lyricist one over Arranger. Thanks for helping me notice :)